libyelp/yelp-uri.c

```diff
@@ -846,20 +857,25 @@ resolve_ghelp_uri (YelpUri *uri)
 static void
 resolve_help_uri (YelpUri *uri)
 {
-    /* help:document[/page][?query][#frag]
+    /* help:[/*]document[/page][?query][#frag]
```

Comment:
> pick [bb41c14](https://github.com/endlessm/yelp/commit/bb41c143338b08fe1a85846a0e5ba4f296b0d106) yelp-uri: Parse extra slashes after colon in resolve_help_uri()
> keep
> 
> Based on the fact that this adds -Wno-error=comment, I think this patch must introduce compiler warnings. Let's fix those warnings, instead.

This wasn't addressed. I've looked a little closer and I'd guess that the problem is on this line, where the string `/*` appears in a comment:
```
libyelp/yelp-uri.c: In function ‘resolve_help_uri’:
libyelp/yelp-uri.c:860:14: warning: "/*" within comment [-Wcomment]
     /* help:[/*]document[/page][?query][#frag]
```

I suggest changing this line to:

```suggestion
    /* help:[/+]document[/page][?query][#frag]
```

and dropping the `-Wno-error=comment` changes to the makefile.

Reply:
Addressed, squashed and pushed.